src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -13,9 +13,6 @@
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <WindowsRID>win</WindowsRID>
     <PackageTargetFramework Condition="'$(TargetsUnix)' == 'true'">netstandard1.4</PackageTargetFramework>
-    <PackageTargetRuntime Condition="'$(TargetsUnix)' == 'true'">unix</PackageTargetRuntime>
-    <!-- we must package netcore50 as RID-specific so that it will override the netstandard impl -->
-    <PackageTargetRuntime Condition="'$(TargetGroup)' == 'netcore50'">$(WindowsRID)</PackageTargetRuntime>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
```

Comment:
unrelated but where are we setting PackageTargetFramework=netstandard1.3 for this project with a default TargetGroup?